spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -29,9 +29,13 @@ and retrieved via the Java-API and also be exposed via the REST-API (see <<archi
 TIP: Implementors of this specification can use the Java API to also expose metrics for _base_ and _vendor_ scope by
 using the respective Metric Registry.
 
-In order to make setting the values easier, Annotations are made available.
+There are two options for registering metrics. The easier one is using annotations - the metrics declared
+by annotations will be automatically added to the registry when the application starts. In some cases, however,
```

Comment:
Should we use bullet points?

Reply:
In this case I'd personally say no, at least it would require quite a bit of rephrasing to do that :) Also with just two items, a bullet point list seems a bit of an overkill to me.